### ISSUE TITLE
Разбор временных объектов

### ISSUE THREAD

**Issue by dveselov** [bureaucratic-labs/natasha#16]
Нужно научить парсер понимать даты и интервалы, с сериализацией в объекты `datetime.datetime`

* https://github.com/olebedev/when
* https://github.com/facebookincubator/duckling
* https://github.com/wit-ai/duckling_old

**Comment by ollmer** [comment]
Скажите, а какие планы по этой фиче? Когда примерно можно ожидать релиза?

**Comment by alexanderkuk** [comment]
Может быть вам подойдёт http://dateparser.readthedocs.io/en/latest/ ?

**Comment by ollmer** [comment]
Пробовал его сегодня, не все кейсы вытягивает (

**Comment by alexanderkuk** [comment]
Какие не вытягивает?

**Comment by ollmer** [comment]
print(dateparser.parse('через 30 секунд', languages=['ru']))
>>None
print(dateparser.parse('5 часов', languages=['ru']))
>>2018-02-07 07:15:57.283900 # построил как "5 часов назад"
print(dateparser.parse('в 5 часов', languages=['ru']))
>>2018-02-07 07:16:35.001085 # построил как "5 часов назад"

**Comment by alexanderkuk** [comment]
Примерно в марте планируется обновление правил Наташи в связи с переездом на новый парсер https://github.com/natasha/yargy/pull/48 . Вы можете составить список примеров строк, которые должны разбираться, как это сделано, например в https://github.com/natasha/natasha/issues/9#issuecomment-276799414 . Мы постараемся их учесть при обновлении

**Comment by ollmer** [comment]
Круто, спасибо!
Я сегодня въезжал в текущий yargy, попробую пока набросать грамматику для времени по аналогии с имеющейся для дат.

**Comment by oserikov** [comment]
Привет! А это живая issue?

**Comment by kuk** [comment]
Да, живая. Нужно тестовые данные. Не получается решать задачу "улучшить извлечение объектов даты и времени". Нужны реальные тексты с датами, временем, которые не парсятся.
